src/index.ts

```diff
@@ -233,7 +233,10 @@ export function loadConfig(configPath: string | undefined): ConfigObject {
     if (!fs.statSync(configPath)) {
       throw new Error("Config file does not exist")
     }
-    config = JSON.parse(fs.readFileSync(configPath, "utf8"))
+    let result = yaml.safeLoad(fs.readFileSync(configPath, "utf8"))
+    if (result != null) {
```

Comment:
We should probably throw an exception if the file parsed without error but returns a null, no?